_docs/backenders/strategy-2021/roles/capacity.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Template
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Capacity role definition
+
+**Current responsible**: -
+
+## Responsibility:
+    **Meetings and other FA:**
+- Facilitator on the capacity part of the BE SMO meeting.
+- Attend to capacity monthly meeting.
+- Participate in fortnightly BD meetings.
+- Participate in BD/FE retrospectives every 6 months.
+- Point of contact for other functional areas.
+
+    **Inside the team:**
+- Having a track on the project proposal pipe.
+- Map and communicate skills.
+- Improve communication of medium and long-term capacity estimations.
+- Ensure alighnment between the capacity needs, hiring and outsourcing processes.
```

Comment:
SOmething like that @agnessa ?

Reply:
I leave it for @aagm to decide, given that she has bravely volunteered :)